src/_premake_init.lua

```diff
@@ -268,6 +286,22 @@
 	}
 
 	api.register {
+		name = "debugtoolargs",
+		scope = "config",
+		kind = "list:string",
+		tokens = true,
+		pathVars = true,
+	}
+
+	api.register {
+		name = "debugtoolcommand",
+		scope = "config",
+		kind = "path",
+		tokens = true,
+		pathVars = true,
+	}
```

Comment:
The NaCl module does set that for you. It just calls the APIs to set a default: https://github.com/TurkeyMan/premake-nacl/blob/master/nacl.lua#L62

The NaCl sets up a lot of defaults via API calls. Mainly to configure the debugger (which is a comprehensive process for NaCl).